### PR TITLE
RFC #3 Adopt Deep Work as a Team Norm

### PR DESCRIPTION
**Pull request by chrisblackburn**

### CODE REVIEW

File: rfc-003-adopt-deep-work-as-team-norm.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- [Deep work at Trello](https://blog.trello.com/deep-work-productivity-interview-with-cal-newport?utm_source=newsletter&utm_medium=email&utm_campaign=december2017_ENnewsletter)
+- [Deep work at Evernote](https://blog.evernote.com/blog/2017/02/23/deep-work-matters-distracted-world/)
+
+## Proposal
+
+An element of shallow work is necessary for most roles within the business. Customers need to be conversed with, emails (somtimes) need to be answered, it's fun and rewarding to chat with peers on Slack.
+
+In their next 121, every member of the team collates a Deep Work plan. Thie involves the team member identifying the tasks that they typically need to perform in the week. The team member then suggests an optimal ratio between deep and shallow work for them to optimally perform their role.
+
+As part of the Delivery Healthcheck that teams will be reporting on from next year, deep work for all members of the team will tracked according to the individuals ideal ratio.
```
Comment (SebAshton) [inline rfc-003-adopt-deep-work-as-team-norm.md]: Delivery Heathcheck? Is this in another RFC or TBC meeting?

File: rfc-003-adopt-deep-work-as-team-norm.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Further reading:
+
+- [Deep work at Trello](https://blog.trello.com/deep-work-productivity-interview-with-cal-newport?utm_source=newsletter&utm_medium=email&utm_campaign=december2017_ENnewsletter)
+- [Deep work at Evernote](https://blog.evernote.com/blog/2017/02/23/deep-work-matters-distracted-world/)
+
+## Proposal
+
+An element of shallow work is necessary for most roles within the business. Customers need to be conversed with, emails (somtimes) need to be answered, it's fun and rewarding to chat with peers on Slack.
+
+In their next 121, every member of the team collates a Deep Work plan. Thie involves the team member identifying the tasks that they typically need to perform in the week. The team member then suggests an optimal ratio between deep and shallow work for them to optimally perform their role.
```
Comment (SebAshton) [inline rfc-003-adopt-deep-work-as-team-norm.md]: `This involves`

File: rfc-003-adopt-deep-work-as-team-norm.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+Further reading:
+
+- [Deep work at Trello](https://blog.trello.com/deep-work-productivity-interview-with-cal-newport?utm_source=newsletter&utm_medium=email&utm_campaign=december2017_ENnewsletter)
+- [Deep work at Evernote](https://blog.evernote.com/blog/2017/02/23/deep-work-matters-distracted-world/)
+
+## Proposal
+
+An element of shallow work is necessary for most roles within the business. Customers need to be conversed with, emails (sometimes) need to be answered, it's fun and rewarding to chat with peers on Slack.
+
+In their next 121, every member of the team collates a Deep Work plan. Thie involves the team member identifying the tasks that they typically need to perform in the week. The team member then suggests an optimal ratio between deep and shallow work for them to optimally perform their role.
```
Comment (craigjbass) [inline rfc-003-adopt-deep-work-as-team-norm.md]: `Thie`

File: rfc-003-adopt-deep-work-as-team-norm.md (hunk 1 above)
Comment (chrisblackburn) [reply rfc-003-adopt-deep-work-as-team-norm.md]: No, I think the Delivery Healthcheck will be straight up Dictatorship

Essentially a means of reporting how much of our Delivery Standards (also a Dictatorship initiative) are being applied to each delivery

File: rfc-003-adopt-deep-work-as-team-norm.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Further reading:
+
+- [Deep work at Trello](https://blog.trello.com/deep-work-productivity-interview-with-cal-newport?utm_source=newsletter&utm_medium=email&utm_campaign=december2017_ENnewsletter)
+- [Deep work at Evernote](https://blog.evernote.com/blog/2017/02/23/deep-work-matters-distracted-world/)
+
+## Proposal
+
+An element of shallow work is necessary for most roles within the business. Customers need to be conversed with, emails (sometimes) need to be answered, it's fun and rewarding to chat with peers on Slack.
+
+In their next 121, every member of the team collates a Deep Work plan. This involves the team member identifying the tasks that they typically need to perform in the week. The team member then suggests an optimal ratio between deep and shallow work for them to optimally perform their role.
```
Comment (RyanMacG) [inline rfc-003-adopt-deep-work-as-team-norm.md]: How do you propose we actually measure any of this? Seems very soggy

Comment (chrisblackburn) [reply rfc-003-adopt-deep-work-as-team-norm.md]: Yeah, agree, and IDK. Wanted to put something up here to see how it sits more widely.

RFC coming to ban the word soggy.

File: rfc-003-adopt-deep-work-as-team-norm.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# Adopt Deep Work as a Team Norm
+
+## Summary
+
+Adopt the discipline of [Deep Work](https://blog.evernote.com/blog/2017/02/23/deep-work-matters-distracted-world/) as a [team norm](https://github.com/madetech/handbook#team-norms).
+
+
+## Problem
+
+Our modern work environment can often make it hard for us to engage in deep work. Between our smartphones, Slack, email, distractions in the office, the doorbell etc. it can tricky to find sufficient space to get uninterrupted periods of focus.
```
Comment (sowasred2012) [inline rfc-003-adopt-deep-work-as-team-norm.md]: it can *be* tricky

File: rfc-003-adopt-deep-work-as-team-norm.md (hunk 4 above)
Comment (sowasred2012) [reply rfc-003-adopt-deep-work-as-team-norm.md]: Down with soggy!

File: rfc-003-adopt-deep-work-as-team-norm.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+- [Deep work at Trello](https://blog.trello.com/deep-work-productivity-interview-with-cal-newport?utm_source=newsletter&utm_medium=email&utm_campaign=december2017_ENnewsletter)
+- [Deep work at Evernote](https://blog.evernote.com/blog/2017/02/23/deep-work-matters-distracted-world/)
+
+## Proposal
+
+An element of shallow work is necessary for most roles within the business. Customers need to be conversed with, emails (sometimes) need to be answered, it's fun and rewarding to chat with peers on Slack.
+
+In their next 121, every member of the team collates a Deep Work plan. This involves the team member identifying the tasks that they typically need to perform in the week. The team member then suggests an optimal ratio between deep and shallow work for them to optimally perform their role.
+
+As part of the Delivery Healthcheck that teams will be reporting on from next year, deep work for all members of the team will tracked according to the individuals ideal ratio.
```
Comment (sowasred2012) [inline rfc-003-adopt-deep-work-as-team-norm.md]: - will *be* tracked
- apostrophe in individual's

File: rfc-003-adopt-deep-work-as-team-norm.md (hunk 4 above)
Comment (sowasred2012) [reply rfc-003-adopt-deep-work-as-team-norm.md]: Lots of optimal in that last sentence.

Comment (sowasred2012) [reply rfc-003-adopt-deep-work-as-team-norm.md]: When you say "identifying the tasks they typically need to perform in the week", are those tasks considered shallow or deep? It reads like we want people to get into deep work mode specifically to tackle regular/monotonous tasks, worth calling out that deep work is good for more than that i.e. putting real thought into solving problems and/or evolving practices.

Comment (sowasred2012) [reply rfc-003-adopt-deep-work-as-team-norm.md]: Not sure how any of it gets measured, but might be interesting for people to note what's distracted them/pulled them out of deep work when distractions happen, to see if there's any recurring themes (bordering on playing a blame game though).

File: rfc-003-adopt-deep-work-as-team-norm.md (hunk 6 above)
Comment (camillavk) [reply rfc-003-adopt-deep-work-as-team-norm.md]: How are we going to be tracking deep work?

**Comment by RyanMacG** [comment]
As an actual comment on the concept rather than the content, I'm still pretty against this. There didn't seem like there was huge appetite coming out of the comradro on this topic and it seems like a very top-down prescription of how teams _should_ work where previously I felt that was left up to them

File: rfc-003-adopt-deep-work-as-team-norm.md (hunk 6 above)
Comment (chrisblackburn) [reply rfc-003-adopt-deep-work-as-team-norm.md]: @camillavk IDK. Requesting comments :-)

**Comment by chrisblackburn** [comment]
@RyanMacG in an RFC to encourage a discussion rather than a prescription :-)

I think there's an interesting questions as to how much responsibility the company has for providing an environment and coaching to enable people to realise their potential. From what I observe and experience I wonder if this could be part of this.

**Comment by RyanMacG** [comment]
Yeah, I guess I'm seeing things like the healthcheck (dictatorship) being a very aggressive implementation of what could otherwise be a useful concept if we were to roadtest it on a smaller scale first. Should have been clearer in the initial comment

**Comment by craigjbass** [comment]
My thoughts are that unless there is a clear understanding of how we're going to track deep work and commit to something tangible, I can't approve this.

> In their next 121, every member of the team collates a Deep Work plan. This involves the team member identifying the tasks that they typically need to perform in the week. The team member then suggests an optimal ratio between deep and shallow work for them to optimally perform their role.

This assumes that it's possible to identify "typical tasks". I think that's quite impossible.

I wonder if this is best applied at a pair level, rather than at a 121 level.

**Comment by chrisblackburn** [comment]
Coming back after a rebrand at a later date

**Pull request closed without merging**